htdocs/helpFiles/Entering-Units.html

```diff
@@ -15,23 +15,23 @@ <h2 style="text-align: center">Units Available in WeBWorK</h2>
 	</tr>
 	<tr>
 		<td style="padding: 0.5rem; border: 1px solid black">Seconds</td>
-		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">s</td>
+		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">s, sec, second or seconds</td>
 	</tr>
 	<tr>
 		<td style="padding: 0.5rem; border: 1px solid black">Minutes</td>
-		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">min</td>
+		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">min, minute or minutes</td>
 	</tr>
 	<tr>
 		<td style="padding: 0.5rem; border: 1px solid black">Hours</td>
-		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">hr</td>
+		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">hr, h, hour or hours</td>
```

Comment:
It also accepts `hrs`. I'm not clear though if you want this to list every variant.